### ISSUE TITLE
Incompatibility between ambient occlusion and parallax occlusion.

### ISSUE THREAD

**Issue by SamuelTS** [pissang/claygl#111]
Hello @pissang,

When I use ambient occlusion **with** parallax occlusion, shadows strangely surround geometry. This looks like a bug:
![ao_with_po](https://user-images.githubusercontent.com/36248671/69577807-7d1e3500-0fcf-11ea-8182-a85e65e739a7.jpg)

Below, when I **only** use parallax occlusion. This is OK:
![po_only](https://user-images.githubusercontent.com/36248671/69577991-d9815480-0fcf-11ea-82c6-c529ce2ca4f4.jpg)

Best regards,

Samuel Tallet

**Comment by pissang** [comment]
@SamuelTS Hi, can you provide your asset for me to debug?

**Comment by SamuelTS** [comment]
Yes, for sure. Thank you! You can download glTF model asset [here](https://mega.nz/#!AMlnRCRQ!ma5jtv4AkVnJX8IpV1rQ1uJVVgVcaPMe9rPtPrRmu_8). Diffuse/normal/AO map is embedded.

And here is parallax occlusion map:

![castle_brick_02_red_disp_4k](https://user-images.githubusercontent.com/36248671/69612005-33fdcd80-102f-11ea-89d5-65508c09f05e.jpg)

Best regards,
Samuel Tallet

**Comment by pissang** [comment]
Thanks @SamuelTS
